docs/source/citation.rst

```diff
@@ -0,0 +1,49 @@
+.. _acknowledgement:
+
+Acknowledgement
+===============
+
+openPMD (Open Standard for Particle-Mesh Data Files) is a community project with many people contributing to it.
+If you use openPMD and/or openPMD related software in your work, please credit it when publishing and/or presenting work performed with it in order to give back to the community.
+
+openPMD-standard
+----------------
+
+The central definition of openPMD is the meta-data schema defined in ... .
+The most general reference to openPMD is:
+
+    Axel Huebl, Remi Lehe, Jean-Luc Vay, David P. Grote, Ivo F. Sbalzarini, Stephan Kuschel, David Sagan, Christopher Mayes, Frédéric Pérez, Fabian Koller, and Michael Bussmann.
+    "openPMD: A meta data standard for particle and mesh based data,"
+    `DOI:10.5281/zenodo.591699 <https://doi.org/10.5281/zenodo.591699>`_ (2015)
+
+Since the openPMD-standard is an actively evolving meta-data schema, a specific version of the openPMD standard might be used in your work.
+You can select a version-specific DOI from the `release page <https://github.com/openPMD/openPMD-standard/releases>`_ and add the version number to the cited title, e.g.
+
+    ... (author list as above) ...
+    "openPMD 1.1.0: A meta data standard for particle and mesh based data," `DOI:10.5281/zenodo.1167843 <https://doi.org/10.5281/zenodo.1167843>`_ (2018)
+
+openPMD-api
+-----------
+
+openPMD-api is a reference implementation of the openPMD-standard for popular data formats.
+
+openPMD-api is a scientific software and cited generally as
+
+    Fabian Koller, Franz Poeschel, Junmin Gu, and Axel Huebl.
+    "openPMD-api: C++ & Python API for Scientific I/O with openPMD"
+    `DOI:10.14278/rodare.27 <https://doi.org/10.14278/rodare.27>`_ (2018)
+
+As a software citation, you almost always want to refer to a specific version of openPMD-api in your works:
+
+Software Version
+~~~~~~~~~~~~~~~~
+
+`Release package page <https://github.com/openPMD/openPMD-api/releases>`_ ...
+Add the version number to the cited title, e.g.
+
+    ... (author list as above) ...
+    "openPMD-api 0.10.3: C++ & Python API for Scientific I/O with openPMD,"
+    `DOI:10.14278/rodare.209 <https://doi.org/10.14278/rodare.209>`_ (2019)
+
+Use the openPMD-api functionality ... to get and document software versions of dependent I/O backends.
+It is good scientific practice to document all software versions in, e.g. a methods section.
```

Comment:
I moved this line to before  this last section. Because, to me, it reads like the motivation for including the section in this documentation page. But I am not a good writer or anything, so feel free to disagree!

Reply:
Sure thing, thanks!